CHANGELOG.md

```diff
@@ -21,6 +24,9 @@
 
 ### Changed
 
+- Bump kedro requirement to <=0.16.5
```

Comment:
We should check that the tests are not broken with kedro<=0.16, because the CI only tests for the last version of kedro, and wa can't be sure that nothing is broken.